kakao/src/main/kotlin/com/agoda/kakao/delegates/impl/ViewInteractionDelegateImpl.kt

```diff
@@ -0,0 +1,54 @@
+package com.agoda.kakao.delegates.impl
+
+import android.support.test.espresso.*
```

Comment:
@Unlimity Do we have any concerns about wildcards import?

Reply:
That is undesirable, but I recommend remove this class, so no matter

Reply:
Why remove this class?

Reply:
You can look up the reason in my review, but generally speaking, default behavior can be implemented directly in the interface through default implementation.

Reply:
Interface `ViewInteractionDelegate` has a property `val viewInteraction: ViewInteraction`. Do you prefer to use anonymous class with this property implementation?

Reply:
If you have declared property in the interface, you can totally use it in default implementations since interface properties does not have a backing field and actually are just getter and setter on the JVM level.
And yep, you can provide it via `object : ViewInteractionDelegate { override val viewInteraction = ... }`

Reply:
ok

Reply:
ok